spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -259,6 +259,7 @@ object SpdxDeclaredLicenseMapping {
         "Public domain (CC0-1.0)" to CC0_1_0.toExpression(),
         "PublicDomain" to licenseRef("public-domain-disclaimer", "scancode"),
         "Python Software Foundation" to PYTHON_2_0.toExpression(),
+        "PSFL" to PYTHON_2_0.toExpression(),
```

Comment:
This entry should be above "Perl Artistic v2" to match the sorting in this file.

Reply:
done.

Reply:
lol, indeed it was `above`.